app/controllers/cases_controller.rb

```diff
@@ -93,13 +93,15 @@ def edit
   def update
     @case = Case::Base.find(params[:id])
     authorize @case
+    @case = @case.decorate
+    if self.class.method_defined?(:persist_session_data)
+      self.persist_session_data(params)
+    end
```

Comment:
I'd rather we weren't editing cases controller like this - wondering if we can have the update method in offender_sar_controller.rb call "persist_session_data" on something there then call `super`